airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/check/commands.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+import logging
+import sys
+from typing import List
+
+import anyio
+import asyncclick as click
+import dagger
+from pipelines.airbyte_ci.format.actions import run_check
+from pipelines.airbyte_ci.format.check.utils import log_output
+from pipelines.airbyte_ci.format.containers import (
+    format_java_container,
+    format_js_container,
+    format_license_container,
+    format_python_container,
+)
+from pipelines.cli.click_decorators import click_ignore_unused_kwargs, click_merge_args_into_context_obj
+from pipelines.models.contexts.click_pipeline_context import ClickPipelineContext, pass_pipeline_context
+
+
+@click.group(
+    help="Run code format checks and fail if any checks fail.",
+    invoke_without_command=True,
+    chain=True,
+)
+@click.option("--list-errors", is_flag=True, default=False, help="Show detailed error messages for failed checks.")
```

Comment:
I think this is the right call. Anyway, in terms of action to take, what's the user going to do? They're going to `format fix` and look at the results with `git diff`, rather than digging for a specific line number in a specific file.